pkg/proxy/iptables/proxier.go

```diff
@@ -190,14 +189,6 @@ func NewProxier(ipt utiliptables.Interface, exec utilexec.Interface, syncPeriod
 		return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlRouteLocalnet, err)
```

Comment:
Worse, there was a separate PR, somewhere, that made the no-op plugin go away and be a nil pointer.